graylog2-server/src/main/java/org/graylog2/plugin/DocsHelper.java

```diff
@@ -17,12 +17,14 @@
 package org.graylog2.plugin;
 
 public enum DocsHelper {
-    PAGE_SENDING_JSONPATH("json"),
-    PAGE_SENDING_IPFIXPATH("ipfix-input"),
-    PAGE_ES_CONFIGURATION("elasticsearch"),
-    PAGE_ES_VERSIONS("elasticsearch#elasticsearch-versions");
+    PAGE_SENDING_JSONPATH("getting_in_log_data/json_path_from_http_api_input.html"),
+    PAGE_SENDING_IPFIXPATH("getting_in_log_data/ipfix_input.html"),
+    PAGE_ES_CONFIGURATION("https://go2docs.graylog.org/current/setting_up_graylog/server.conf.html#OpenSearch"),
+    PAGE_ES_VERSIONS("downloading_and_installing_graylog/installing_graylog.html#CompatibilityMatrix"),
+    REPORTING_HELP("interacting_with_your_log_data/reporting.html");
 
-    private static final String DOCS_URL = "https://docs.graylog.org/docs/";
+    private static final String SERVER = "https://go2docs.graylog.org";
+    private static final String VERSION = "current";
```

Comment:
Should we tie this to the current server's version?

Reply:
I wanted to keep this change small, only fixing the links and the new host. I added the separate `version` string because a version is needed to link correctly and I thought it was good to already have it as it's own entity for further improvements in the future (linking to the server-version) but not going all the way in this PR.

Reply:
included your suggestion